utils/ankiexport.py

```diff
@@ -80,20 +88,92 @@ def export_card(self, media_file, audio_track, text_primary, text_secondary, tim
             return -2
 
         return 0
-        
 
-
-    def make_audio(self, media_file, audio_track, start, end, out_path):
+    def ffmpeg_audio(self, media_file, audio_track, start, end, out_path):
+        args = [
+                'ffmpeg',
```

Comment:
This should use the ffmpeg executable from https://github.com/migaku-official/migaku-mpv/blob/063fe424754af72350d2ad6935c5bd57d35a1d3d/migaku_mpv.py#L783